version/version.go

```diff
@@ -38,7 +38,7 @@ var Version = semver.MustParse("3.1.0-dev")
 var APIVersion = map[Tree]map[Level]semver.Version{
 	Libpod: {
 		CurrentAPI: Version,
-		MinimalAPI: semver.MustParse("3.0.0"),
+		MinimalAPI: semver.MustParse("3.2.0-dev"),
```

Comment:
I don't think we have to bump the minimal API version. @jwhonce PTAL

Reply:
The tests start blowing up if we don't - the Current API version is set to v3.2.0-dev, and according to semantic versioning, v3.2.0 is more recent than v3.2.0-dev (suffixes indicate pre-release). Unless we bump this. the minimum API version is lower than the current API version.

Reply:
I don't think so, looking at the changes I can see that you changed this from 3.0.0 so it worked together with the 3.1.0-dev version.

Reply:
@Luap99 @mheon I would suggest that MimimalAPI be n-1 (Current release version) at least at until the end of the release cycle for a given branch. As we stabilize the API, I could see a day where the drift between Current and Minimal grows.